source/frontend/src/components/propertySelector/MapSelectorContainer.tsx

```diff
@@ -55,9 +55,16 @@ export const MapSelectorContainer: FunctionComponent<IMapSelectorContainerProps>
         }
         const pid = pidFromFeatureSet(property);
         const pin = pinFromFeatureSet(property);
-        if (!pid || !pin) {
+        if (!pid && !pin) {
           return property;
         }
+        const queryObject = {};
+        if (pid.length > 0) {
+          queryObject['PID'] = pid;
+        }
+        if (isValidId(+pin)) {
+          queryObject['PIN'] = pin;
+        }
```

Comment:
Could you please comment what we want to do with this **queryObject**? It seems it gets created but not used anywhere. Perhaps it is meant to be used in the line below (line 68)?